code/modules/mob/living/carbon/human/species.dm

```diff
@@ -66,6 +66,8 @@
 	var/attack_verb = "punch"	// punch-specific attack verb
 	var/sound/attack_sound = 'sound/weapons/punch1.ogg'
 	var/sound/miss_sound = 'sound/weapons/punchmiss.ogg'
+	
+	var/knockout = /obj/item
```

Comment:
The reason for this *was* line 1095 where you forgot to use `I.knockout` and instead used `knockout` (you have now fixed this, so this line no longer necessary, because it's not used anymore).